frontend/app/templates/components/team-show.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="container mt-1">
+  <div class="accordion" id={{concat "teamAccordion-" @team.id}}>
+    <div class="row mb-10">
+      <div role="button" data-toggle="collapse" data-target={{concat "#teamCollapse-" @team.id}} aria-expanded="true" aria-controls={{concat "teamCollapse-" @team.id}}>
+        <h1 class="d-inline">{{@team.name}}</h1>
+        <span class="ml-5">
+          <LinkTo @route="teams.edit" @model={{@team}}>
+            <img class="icon-button" src="/ember/assets/images/edit.svg" alt="edit">
+          </LinkTo>
+          <LinkTo @route="teams.configure" @model={{@team.id}}>
+            <img class="icon-button" src="/ember/assets/images/team_config.svg" alt="configure">
+          </LinkTo>
+        </span>
+      </div>
+      <hr>
+    </div>
+    <div id={{concat "teamCollapse-" @team.id}} class="collapse" aria-labelledby={{concat "#teamHeading-" @team.id}} data-parent={{concat "#teamAccordion-" @team.id}}>
+      <div>
+        <p>{{@team.description}}</p>
+      </div>
+      {{#each @team.folders as |folder|}}
+        <FolderShow @folder={{folder}}/>
+      {{/each}}
+    </div>
```

Comment:
Afer rebasing where the Bootstrap Addon was added, you can use the BSCollapse here like in _nav-bar.hbs_